src/miner.cpp

```diff
@@ -61,6 +62,30 @@ class COrphan
     }
 };
 
+//!
+//! \brief Helper: update the miner status with a message that indicates why
+//! \c CreateCoinStake() skipped a cycle and then return \c false .
+//!
+//! \param status  The global miner status object to update.
+//! \param message Describes why the wallet contains no coins for staking.
+//!
+//! \return Always \false - suitable for returning from the call directly.
+//!
+bool BreakForNoCoins(CMinerStatus& status, const char* message)
```

Comment:
This function looks like suitable for inclusion into CMinerStatus class.

Reply:
We could, but it's an implementation detail of the miner loop. I don't think we want code outside this file to call that function. It just makes the early return conditions in the miner loop more readable/maintainable.

Reply:
Let's leave it this way for now. We can fancy it up in Elizabeth if we want.